public/js/index.js

```diff
@@ -1,7 +1,22 @@
-(() => {
```

Comment:
it would be better if you change this file name to functions.js :smile_cat:

Reply:
ok :+1:

Reply:
done :+1: